specification/authorization/resource-manager/Microsoft.Authorization/preview/2024-02-01-preview/RoleManagementPolicyAssignment.json

```diff
@@ -0,0 +1,393 @@
+          },
+          "readOnly": true,
+          "description": "The readonly computed rule applied to the policy."
+        },
+        "policyAssignmentProperties": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Additional properties of scope, role definition and policy",
+          "$ref": "#/definitions/PolicyAssignmentProperties"
+        }
+      },
+      "type": "object",
+      "description": "Role management policy assignment properties with scope."
+    },
+    "RoleManagementPolicyAssignmentListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoleManagementPolicyAssignment"
+          },
+          "description": "Role management policy assignment list."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "type": "object",
+      "description": "Role management policy assignment list operation result."
+    },
+    "PolicyAssignmentProperties": {
+      "properties": {
+        "scope": {
+          "type": "object",
+          "description": "Details of the resource scope",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Scope id of the resource"
+            },
+            "displayName": {
+              "type": "string",
+              "description": "Display name of the resource"
+            },
+            "type": {
+              "type": "string",
+              "description": "Type of the resource"
+            }
+          }
+        },
+        "roleDefinition": {
+          "type": "object",
+          "description": "Details of role definition",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Id of the role definition"
+            },
+            "displayName": {
+              "type": "string",
+              "description": "Display name of the role definition"
+            },
+            "type": {
+              "type": "string",
+              "description": "Type of the role definition"
+            }
+          }
+        },
+        "policy": {
+          "type": "object",
+          "description": "Details of the policy",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Id of the policy"
+            },
+            "lastModifiedBy": {
+              "$ref": "./common-types.json#/definitions/Principal"
+            },
+            "lastModifiedDateTime": {
+              "type": "string",
+              "format": "date-time",
+              "description": "The last modified date time."
+            }
+          }
+        }
+      },
+      "type": "object",
+      "description": "Expanded info of resource scope, role definition and policy"
+    },
+    "CloudError": {
```

Comment:
You can get rid of this and CloudErrorBody if you use the common types definition.